src/component/parser/json/converter/ProcessConverter.ts

```diff
@@ -222,10 +228,10 @@ export default class ProcessConverter {
     }
 
     if (!ShapeUtil.isCallActivity(kind)) {
-      // eslint-disable-next-line @typescript-eslint/ban-ts-comment -- the 'instantiate' property doesn't always exist, in that case, it will be undefined
-      // @ts-ignore
-      return new ShapeBpmnActivity(bpmnElement.id, bpmnElement.name, kind, parentId, bpmnElement.instantiate, markers);
+      const instantiate = 'instantiate' in bpmnElement ? (bpmnElement.instantiate as boolean) : undefined;
```

Comment:
**suggestion:** introduce a function in this module to compute the instantiate value. This will remove the duplication and will allow to inline the call to this new function in the Shape constructor call.